packages/lambda-xyz/src/routes/tile.ts

```diff
@@ -141,7 +133,10 @@ export async function Wmts(req: LambdaContext, wmtsData: TileDataWmts): Promise<
 
     const host = Env.get(Env.PublicUrlBase);
 
-    const xml = buildWmtsCapability(host, req.apiKey || '', wmtsData.tileSet, wmtsData.projection);
+    // TODO get all projections if wmtsData.projection is null
+    await loadTileSet(req, wmtsData.name, wmtsData.projection ?? EPSG.Google);
```

Comment:
could we make this return a tileset[] and push that into buildWmtsCapability?

Reply:
Eventually but not needed now so don't want to future proof it until we support another projection; we may end up changing the way we do things

Reply:
Ahh was more thinking there are a number of `.get(name, projection)` that assume that a `.load()` has been run inside of non async functions.

this could lead to confusion down the line if the function is called before a `.load()` is called.

if this `loadTileSet()` returns a `TileSet` it then can be passed into the function and it is already initialized.

It doesnt have to reutrn a TileSet[] but could just return TileSet then pass that into `buildWmtsCapability`

Reply:
Now just does 3857 if missing and passes TileSet to wmts. In Future will pass an array of TileSets